hyperspy/io_plugins/bcf.py

```diff
@@ -1039,53 +1047,57 @@ def bcf_hyperspectra(obj_bcf, index=0, downsample=None, cutoff_at_kV=None):
+                               'offset': 0,
+                               'scale': obj_bcf.hypermap[index].xcalib,
+                               'units': 'µm'},
+                              {'name': 'Energy',
+                               'size': obj_bcf.hypermap[index].hypermap.shape[2],
+                               'offset': obj_bcf.hypermap[index].calib_abs,
+                               'scale': obj_bcf.hypermap[index].calib_lin,
+                               'units': 'keV'}],
+                     'metadata':
+                     # where is no way to determine what kind of instrument was used:
+                     # TEM or SEM
+                     {'Acquisition_instrument': {
+                         mode: {
+                             #'beam_current': 0.0,  # There is no technical
+                             # possibilities to get such parameter from bruker
+                             # or some SEM's'
+                             'beam_energy': obj_bcf.header.sem.hv,
+                             'tilt_stage': obj_bcf.header.stage.tilt_angle,
+                             'stage_x': obj_bcf.header.stage.x,
+                             'stage_y': obj_bcf.header.stage.y,
+                             'magnification': obj_bcf.header.sem.mag,
+                             'Detector': {
+                                 'EDS': {
+                                     'azimuth_angle': eds_metadata.azimutAngle,
+                                     'elevation_angle': eds_metadata.elevationAngle,
+                                     'detector_type': eds_metadata.detectorType
                                  }
+                             }
                          }
-                                         },
-              'General': {'original_filename': obj_bcf.filename.split('/')[-1],
-                          'title': 'EDX',
-                          'datetime': obj_bcf.header.datetime},
-              'Sample': {'name': obj_bcf.header.name,
-                         'elements': list(obj_bcf.header.elements),
-                         'xray_lines': gen_elem_list(obj_bcf.header.elements)},
-              'Signal': {'signal_type': 'EDS_SEM',
-                         'record_by': 'spectrum'},
-             }
-           }]
+                     },
+        'General': {'original_filename': obj_bcf.filename.split('/')[-1],
+                         'title': 'EDX',
+                         'date': obj_bcf.header.date,
+                         'time': obj_bcf.header.time},
+        'Sample': {'name': obj_bcf.header.name,
+                         'elements': sorted(list(obj_bcf.header.elements)),
+                         'xray_lines': sorted(gen_elem_list(obj_bcf.header.elements))},
+        'Signal': {'signal_type': 'EDS_%s' % mode,
+                         'record_by': 'spectrum',
+                         'quantity': 'X-ray intensity (counts)'},
```

Comment:
I think "electron intensity" is quite good for EELS, although this is not the most widely used and may sound a bit unusual... but most of the people (including myself) are a bit lazy when making figure and we end up with EELS signal, EELS intensity (a. u.) or even intensity (a. u.) on its own.
EELS intensity with either "counts" or "electrons" as units (depending, if the conversion factor is used or not) will be also good.